src/sentry_clj/core.clj

```diff
@@ -195,6 +196,8 @@
 
      (.setDsn sentry-options dsn)
 
+     (when external-configuration
+       (.setEnableExternalConfiguration sentry-options external-configuration))
```

Comment:
Could you rename the key to be `enable-external-configuration` for consistency with the other `enable` please (and update the documentation).

Other than that, looks good.

Thank you.

Reply:
Sure thing!

Reply:
Fixed in cad20fb